src/client/cl_predict.c

```diff
@@ -102,16 +107,16 @@ static void Cl_ClipTraceToEntities(cl_trace_t *trace) {
 			continue;
 		}
 
-		if (s->number == cl.client_num + 1) {
-			continue;
-		}
-
```

Comment:
This was redundant, as the "self-check" is now handled by the cgame with `Cg_Self` and `cl.entity` (see below).